docs/develop/localization.md

```diff
@@ -145,9 +145,106 @@ For Outlook add-ins, the [SourceLocation] element also aligns to the form factor
 </PhoneSettings>
 ```
 
+## Localize extended overrides
+
+Some extensibility features of Office Add-ins, such as keyboard shortcuts, are configured with JSON files that are hosted on your server, instead of with the add-in's XML manifest. This section assumes that you're familiar with extended overrides. See [Work with extended overrides of the manifest](extended-overrides.md) and [ExtendedOverrides](../reference/manifest/extendedoverrides.md) element.
+
+se the `ResourceUrl` attribute of the [ExtendedOverrides](../reference/manifest/extendedoverrides.md) element to point Office to a file of localized resources. The following is an example.
+
+```xml
+    ...
+    </VersionOverrides>  
+    <ExtendedOverrides Url="https://contoso.com/addin/extended-overrides.json" 
+                       ResourceUrl="https://contoso.com/addin/my-resources.json">
+    </ExtendedOverrides>
+</OfficeApp>
+```
+
+The extended overrides file then uses tokens instead of strings. The tokens name strings in the resource file. The following is a simple example that assigns a keyboard shortcut to function (defined elsewhere) that displays the add-in's task pane. Note about this markup:
```

Comment:
@Rick-Kirkham Can we say something like: "The following is an example that assigns a keyboard shortcut to a function..."

Reply:
done